backend/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,13 @@
+#   local values
```

Comment:
Maybe is better to use ${FIELD_NAME} for specific values that could be different for yours local env

Reply:
That makes sense. So we would all add environment variables to the run configuration/env vars instead.

Although it might be easier to do the same thing we did with the <code>.env</code> file: .gitignore this file and have a <code>application-local.properties.example</code> with all the necessary variables defined as blank. Then everyone would have make their own <code>application-local.properties</code>, or copy someone else's.

Reply:
That makes sense. So we would all add environment variables to the run configuration/env vars instead.

Although it might be easier to do the same thing we did with the <code>.env</code> file: .gitignore this file and have a <code>application-local.properties.example</code> with all the necessary variables defined as blank. Then everyone would have make their own <code>application-local.properties</code>, or copy someone else's.

Reply:
As you wish, i will leave that part to you to decide

Reply:
I'll take a look.

Reply:
I can't really get this library to work - looks like <code>application.properties</code> keeps pulling vars from <code>.env</code> in root no matter what.

I wanted to just rename <code>application-local.properties</code> to <code>.env</code> and pull values from there (when running locally).